### ISSUE TITLE
Block Editor not loading for Simple site

### ISSUE THREAD

**Issue by RealCarFax** [Automattic/wp-calypso#55505]
### Steps to Reproduce

1. Start on https://wordpress.com/home/
2. Click on Pages (Sivut)
3. Click on Edit (Muokkaa) under one of the pages or alternatively click on Add New (Lisää uusi)

### What I expected to happen

The block editor should load

### What actually happened

Only a white page loads, the favicon and site title appear in the browser tab, but nothing else beyond that.

### Context

Customer report on their site.

This issue appears to only occur for Pages and using the Block editor. If we try to edit or create a post, the editor loads properly. If we try to use the classic editor to either edit or create a new page, the editor loads properly that way too.

### Operating System

macOS

### OS Version

Big Sur (11.5)

### Browser

Firefox

### Browser Version(s)

91.0, occurs on other browser as well as user was using Chrome

### Is this specific to applied theme? If so, what is the theme name?

No, occurs regardless of theme.

### Simple/Atomic

Simple

### Console and/or error logs

Here's a screenshot of the console errors I saw: https://d.pr/i/RDsrlM

### Number of Users Impacted

One user

### Available Workarounds

They can use the classic editor but it appears that most of their pages were built with the block editor so that is less than ideal.

### Reproducibility

Never

### Other information

For a whole list of the things we have already tried, please see this thread: https://a8c.slack.com/archives/C03TY6J1A/p1628885776102200

Follow-up Ticket here: #4220624-zen

**Comment by allanftd** [comment]
User followed up on email since they have an event today. Suggested workaround - check if page editor works on the app. 4220624-zen

**Comment by synora** [comment]
I'm able to replicate this from my end (both in and outside a Support Session in the user's account). 

Screenshot of console errors: 
<img width="1617" alt="Screen Shot on 2021-08-17 at 11-31-48" src="https://user-images.githubusercontent.com/10015451/129755872-3f83b39a-4db2-4109-9f63-5f51f3128cf7.png">

I'm using Chrome Version 92.0.4515.107 (Official Build) (x86_64).
My machine: MacOS BigSur v 11.4.

**Comment by dpasque** [comment]
Tagging @Automattic/flow-patrol-create based on issue content

**Comment by yansern** [comment]
For that particular site, the post/page ID 18 is causing out of memory errors. Something to do with a paragraph block if I'm not mistaken. 
<img width="1344" alt="2021-08-20_17-59-42" src="https://user-images.githubusercontent.com/1287077/130261630-e2b38121-aacb-4e73-80e5-56c0c1edd854.png">

**Comment by ivan-ottinger** [comment]
Turns out the content of the page with ID 34 is repeated over and over again (hundreds of times or so) which is likely resulting in those memory errors on other pages as well.

Not sure how this could have happened in the first place, but I believe that if we can somehow cut out the extra lines of content from the page with ID 34, all should be back to normal.

Maybe we could run some DB query, or recreate the page (making sure there isn't repeated content) and then force-delete the current large page.

**Comment by ivan-ottinger** [comment]
I have created a new post draft with the selected content of the broken page (ID 34). By “selected content” I mean that I left out all the repeated content.

The draft can be found in `Posts → Drafts` and it has a title "Page draft".

The reason I created the page as a post is that the block editor isn’t working properly for the user.

We can ask the user to check if the new draft contains everything expected. If yes, then we can try to delete the problematic page.
